src/core/CMakeLists.txt

```diff
@@ -167,21 +167,28 @@ IF (ANDROID)
     androidviewstatus.cpp
     androidprojectsource.cpp
   )
+ELSEIF (${CMAKE_SYSTEM_NAME} STREQUAL "iOS")
+  SET(QFIELD_CORE_HDRS ${QFIELD_CORE_HDRS}
+    ios/iosplatformutilities.h
+  )
+  SET(QFIELD_CORE_SRCS ${QFIELD_CORE_SRCS}
+    ios/iosplatformutilities.mm
+  )
 ENDIF (ANDROID)
 
 FIND_PACKAGE(Sqlite3)
 
-ADD_LIBRARY(qfield_core SHARED ${QFIELD_CORE_SRCS} ${QFIELD_CORE_HDRS})
+ADD_LIBRARY(qfield_core STATIC ${QFIELD_CORE_SRCS} ${QFIELD_CORE_HDRS})
```

Comment:
Always static?

Reply:
that's what @OlivierLDff wrote. is this an issue?

Reply:
Probably not. And if it is, then we can switch again.

Reply:
I would say, always use static by default:
- This remove one step for deployment
- Only the required symbols are embedded making the app smaller

I feel we are in era of packaging everything and creating separate environment for every application.